Page.php

```diff
@@ -167,6 +172,10 @@ function($matches) {return $matches[1] . '{{Cite journal | id=' . $matches[2] .
                       function($matches) {return $matches[1] . '{{Cite journal | doi=' . $matches[2] . ' | ' . strtolower('CITATION_BOT_PLACEHOLDER_BARE_URL') .'=' . base64_encode($matches[2]) . '}}' . $matches[3] ;},
                       $this->text
                       );
+     if (
+        ($ref_count < 2) ||
+        (($citation_count/$ref_count) >= 0.5)
+     ) {
      $this->text = preg_replace_callback(   // like <ref>John Doe, [https://doi.org/10.1244/abc Foo], Bar 1789.</ref>
                                             // also without titles on the urls
```

Comment:
The case without title, where there's only a link in the ref, should not be within this condition.